airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigPersistence.java

```diff
@@ -31,4 +32,7 @@
 
   void loadData(ConfigPersistence seedPersistence) throws IOException;
 
+  // todo (lmossman) - delete this method after the faux major version bump
+  Set<String> getConnectorReposInUse() throws IOException;
```

Comment:
do we need to expose this in the interface? if it's just used in `ServerApp`, I think the configPersistence is actually of type `DatabaseConfigPersistence` can't we just make that method public on that class instead of exposing it on this face. I think it'll save you a bunch of code changes.

Reply:
Ah great catch, you're right I don't need to expose it in the interface. Will just add to DatabaseConfigPersistence!